docs/expo.md

```diff
@@ -0,0 +1,163 @@
+- [Guide for Bare React Native apps using Expo modules](#guide-for-bare-react-native-apps-using-expo-modules)
+
+# Guide for Expo apps
+
+## Overview
+Expo projects can use both third-party React Native libraries with native code and your own custom native code. Creating a development build with Expo CLI allows you to include your specific native dependencies and customizations.
+
+When initializing the Adobe Mobile SDKs, it is required to initialize the SDKs in the native code inside your `AppDelegate` file for iOS and the `MainApplication` file for Android.
+
+By default, Expo projects use Continuous Native Generation (CNG). This means that projects do not have android and ios directories containing the native code and configuration. You can opt out of CNG and directly manage the code and configuration inside your android and ios directories.
+
+## Installation
+
+To generate these directories, run 
+```bash
+npx expo prebuild
+```
+Or compile your app locally.
+```bash
+# Build your native Android project
+npx expo run:android
+# Build your native iOS project
+npx expo run:ios
+```
+
+## Install Adobe Mobile SDKs
+
+Follow the [Installation Guide](../README.md#Installation) to install Adobe SDKs after the android and ios directories are generated.
+
+## Initialize Adobe Mobile SDKs
+Initializing the SDK should be done in native code inside your `AppDelegate` (iOS) and `MainApplication` (Android). The following code snippets demonstrate how to install and register the AEP Mobile Core and Edge Network extensions. Documentation on how to initialize each extension can be found in _./packages/{extension}/README.md_.
+
+##### **iOS**
+
+```objective-c
+//AppDelegate.h
+@import AEPCore;
+@import AEPServices;
+@import AEPLifecycle;
+@import AEPSignal;
+@import AEPEdge;
+@import AEPEdgeIdentity;
+@import AEPEdgeConsent;
+...
+```
+
+```objective-c
+//AppDelegate.m
+...
+@implementation AppDelegate
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
+{
+  self.moduleName = @"AEPSampleApp";
+  self.initialProps = @{};
+  
+  [AEPMobileCore setLogLevel:AEPLogLevelTrace];
+  [AEPMobileCore configureWithAppId:@"YOUR-APP-ID"];
+  const UIApplicationState appState = application.applicationState;
+  [AEPMobileCore registerExtensions:@[
+      AEPMobileLifecycle.class, AEPMobileIdentity.class,
+      AEPMobileEdgeIdentity.class, AEPMobileEdge.class,
+      AEPMobileEdgeConsent.class, AEPMobileEdgeBridge.class
+    ]
+  completion:^{
+    if (appState != UIApplicationStateBackground) {
+      [AEPMobileCore lifecycleStart:nil];
+      }
+      }];
+
+  return [super application:application didFinishLaunchingWithOptions:launchOptions];
+}
+
+@end
+
+```
+
+> To enable the Lifecycle metrics, [implement the Lifecycle APIs](./packages/core/README.md#lifecycle)
+
+> Hint : While running iOS application after Adobe Experience Platform SDK installation. If you have build error that states:
```

Comment:
> [!TIP]
> If the following build error occurs when running the iOS application after Mobile SDK installation:
>
> ```
> ld: warning: Could not find or use auto-linked library 'swiftCoreFoundation'
> ```
>
> This is because the Mobile SDK now requires the app to use Swift interfaces. Adding a placeholder `.swift` file to the project embeds the Swift standard libraries. See the SampleApp in this repository for an example.